src/ape/api/address.py

```diff
@@ -54,6 +54,18 @@ def address(self) -> AddressType:
             :class:`~ape.types.AddressType`
         """
 
+    def __eq__(self, __o: object) -> bool:
+        """
+        Compares AddressAPI/str objs
```

Comment:
```suggestion
        Compares :class:`~ape.api.AddressAPI`/``str`` objects.
```

Reply:
I think this needs to be expanded also to say that it does a string comparison on the address or something.